tests/stub/authorization/test_authorization.py

```diff
@@ -773,11 +784,12 @@ def tearDown(self):
 
     def get_vars(self):
         return {
-            "#VERSION#": "4.0"
+            "#VERSION#": "4.4"
         }
 
     @driver_feature(types.Feature.OPT_AUTHORIZATION_EXPIRED_TREATMENT)
-    def test_should_drop_connection_after_AuthorizationExpired(self):  # noqa: N802,E501
+    def test_should_drop_connection_after_AuthorizationExpired(
+        self):  # noqa: N802,E501
```

Comment:
`# noqa: N802,E501` disables two linter checks
 * `E501` check line length (<= 79 characters)
 * `N802` checks that methods use snake case.
 
 I was sorta ok with this method using some Pascal case because that's the name of the error. I wouldn't mind, however, if it was changed to all snake case. Might be better, stylistically speaking.  The line was not too long until I added the `# noqa: N802`. Since this comment cannot go on another line than the one for which it should disable the check, I also had to tell the linter to allow the over-long line.